google/cloud/storage/examples/storage_bucket_samples.cc

```diff
@@ -936,14 +936,17 @@ void LockRetentionPolicy(google::cloud::storage::Client client, int& argc,
                 << ", status=" << original.status() << std::endl;
       return;
     }
-    StatusOr<void> status = client.LockBucketRetentionPolicy(
+
+    StatusOr<gcs::BucketMetadata> updated = client.LockBucketRetentionPolicy(
         bucket_name, original->metageneration());
-    if (!status.ok()) {
+    if (!updated.ok()) {
```

Comment:
Add a new line before this one to separate the `LockBucketRetentionPolicy()` request and `if (!updated.ok()) {`.

Reply:
Whoops, missed one, `updated` to `updated_metadata` or `updated_meta` if you want to keep the that separate.

Reply:
Added the newline, thanks.

I changed the variable name, but I disagree.  What else could it be given the variable type (which is right there, a couple of lines before)?  I cannot be a `updated_int` or `updated_string`, or `updated_object_metadata`.  The type is `StatusOr<BucketMetadata>` it *must* be a status or a bucket metadata, nothing else will do.

Reply:
Done.